### PR TITLE
build: Add EPUB support

### PR DESCRIPTION
**Pull request by jtojnar**
You can get it by running `nix-build release.nix -A pills.epub`.

Fixes: https://github.com/NixOS/nix-pills/issues/31

### CODE REVIEW

**Comment by spikespaz** [comment]
Why has this been sitting here sine 2021?

**Comment by fricklerhandwerk** [comment]
@spikespaz The Pills don't get much love, and honestly I think they should be overhauled anyway.

@domenkozar @edolstra @garbas can any of you give me merge access so I can take care of things like these?

**Comment by turboMaCk** [comment]
Also there are not that many committers for this repository.

**Comment by jtojnar** [comment]
Simplified the build command: <code>nix-build release.nix -A <del>pills.</del>epub</code>

File: README.md [hunk 1]
```diff
@@ -2,19 +2,12 @@
 
 https://nixos.org/nixos/nix-pills/
 
-## Development
-
- - List of Elements: http://tdg.docbook.org/tdg/5.2/chunk-part-d6252e15351.html
- - Tracking Issue: https://github.com/NixOS/nixpkgs/issues/27908
+You can also built them locally: `nix-build release.nix -A html-split && firefox result/share/doc/nix-pills/index.html`
+And there is also an EPUB version: `nix-build release.nix -A epub && foliate result/share/doc/nix-pills/nix-pills.epub`
```
Comment (fricklerhandwerk) [inline README.md:6]:
```suggestion
You can also built them locally:

    nix-build release.nix -A html-split && firefox result/share/doc/nix-pills/index.html
    
And there is also an EPUB version

    nix-build release.nix -A epub && foliate result/share/doc/nix-pills/nix-pills.epub
```

**Review by fricklerhandwerk — APPROVED** [review]

**Pull request merged**